plexpy/graphs.py

```diff
@@ -1169,15 +1265,17 @@ def get_stream_type_by_top_10_users(self, time_range='30', y_axis='plays', user_
 
         return output
 
-    def _make_user_cond(self, user_id):
+    def _make_user_cond(self, user_id, cond_prefix=None):
```

Comment:
Make the default argument `'AND'` so you don't need to check for `None`.
```suggestion
    def _make_user_cond(self, user_id, cond_prefix='AND'):
```

Reply:
Good point.